spec/views/batch_contexts/index.html.erb_spec.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+RSpec.describe 'batch_contexts/index.html.erb' do
+  let!(:batch_contexts) { create_list(:batch_context_with_deleted_output_dir, 2) }
+  let!(:batch_context_with_globus) { create(:batch_context_with_deleted_output_dir, :with_globus_destination) }
+  let(:globus_url) { "https://app.globus.org/file-manager?&amp;destination_id=endpoint_uuid&amp;destination_path=#{batch_context_with_globus.globus_destination.destination_path}" }
```

Comment:
I'm not sure how the trait is working here, but I think you may be able to use `batch_context_with_globus.globus_destination.url` instead of constructing the URL yourself?

Reply:
yes, you are right and i will use that (though there is a slight formatting tweak needed to match the rendered output)